client/src/components/Home.jsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
JSX transform no longer require React to be in scope.

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform